src/racing/actors.c

```diff
@@ -2247,7 +1154,7 @@ void place_course_actors(void) {
     gNumPermanentActors = gNumActors;
 }
 
-void func_8029E158(void) {
+void load_common_texture(void) {
```

Comment:
This one is going to be a little tough. Because at the very end of the function they yolo'd some calls in there.

`init_actors_and_load_textures`?

Maybe add a comment like this:
```c
/**
 * Calls to init_course_vehicles, and place_course_actors();
 * Loads actor textures, course specific actor textures.
 */
```
Something like that would help I think? I checked the caller/callee graph and this function is ran just after setup_race(); so it's one of the important ones despite that loading textures isn't that important.

Reply:
it load also gTextureFinishLineBanner1 so not only actor

Reply:
Loads a variety of textures; actors, course specific actors, and some common to all courses.